src/emulator/storage/apis/firebase.ts

```diff
@@ -146,9 +147,9 @@ export function createFirebaseEndpoints(emulator: StorageEmulator): Router {
   // list object handler
   firebaseStorageAPI.get("/b/:bucketId/o", async (req, res) => {
     const maxResults = req.query.maxResults?.toString();
-    let response: ListResponse;
+    let listResponse: ListObjectsResponse;
     try {
-      response = await storageLayer.handleListObjects({
+      listResponse = await storageLayer.listObjects({
```

Comment:
Do you mind refactoring [this method](https://github.com/firebase/firebase-tools/blob/tonyjhuang/list/src/emulator/storage/files.ts#L469) to remove the `handle` prefix since you're doing it for the other methods

Reply:
good catch. done.